APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,235 @@
+
+On a longer term horizon, a criteria-based review checklist will allow the
+review process to change to a more peer-review model.  That is, instead of
+every package being vetted by the coordination committee, the committee can act
+more as editors, sometimes offering the review opportunity to other members of
+the Astropy community.  The details of this process will not be set in stone by
+this APE, as they should be flexible enough to accomodate the experiences of the
+first rounds of reviewers.  But this APE sets some initial steps (see
+"Implementation" below) and makes the general idea the eventual goal for the
+affiliated package review process.
+
+
+Branches and pull requests
+--------------------------
+
+N/A
+
+
+Implementation
+--------------
+
+The changes outlined in this APE will be implemented as three distinct efforts:
+
+* The new package review criteria have already been implemented by the
+  coordination committee and reviewed by the community (Available in Google doc
+  form `here <https://docs.google.com/document/d/15PJf2PROXMa7xwTDvWnjXB_9KNuO2Ia4_kkxJ7MPazE/edit?usp=sharing>`_.
+  These criteria will need to be translated to HTML and put on the affiliated
+  package instructions section of the Astropy website.  The affiliated package
+  listing page will also need to be updated for these new criteria - a concept
+  for this new site is `viewable here <example_affilpkg_page.png>`_.
+* The Astropy web site (and to a lesser extent, affiliated package-related docs)
+  will need to be updated to reflect the existence of the coordinated
+  packages.  Additionally, the roles web site will need to be populated with the
+  new roles for the coordinated packages (and the role descriptions).  This
+  requires relatively little effort and can be completed by the Astropy web site
+  maintainers upon acceptance of this APE.
+* A "peer-review" process will need to be set up to enable reviews by community
+  members beyond the coordination committee.  Such processes exist in
+  other code contexts (e.g. the
+  `Journal of Open Source Software <http://joss.theoj.org/>`_ and
+  `Ropensci <https://ropensci.org/>`_) that may have tools that can be re-used
+  to make the process smooth and efficient.  Setting such tools up is likely to
+  take some time, however, and needs to be adaptable enough to respond to the
+  first few reviews.  Hence this APE does not describe the details of this
+  implementation - that is explicitly left to be developed by the Astropy
+  community with the cooordination committee acting as "editors".  However,
+  this APE recommends a simple *starting* point: reviewers can be selected by
+  the coordination committee from the pool of existing affiliated package
+  authors or Astropy core team members.  They receive a prompt to review a new
+  package via email (along with the criteria described a above), and provide
+  their reviews also via email.  The coordination committee will then perform
```

Comment:
reviews by email is too specific - maybe the reviews will be by GitHub. Maybe you could say that this is not a recommended starting point but an example? Or just say that the technological details are beyond the scope but the main thing is just the existence of reviewers as opposed to just the coordination committee.

Reply:
Good point.  Will be a bit less specific on that point.